Vagrantfile

```diff
@@ -1,8 +1,44 @@
 Vagrant.configure("2") do |config|
-  config.vm.box = "fedora/30-cloud-base"
-  config.vm.provision "shell", path:"scripts/bootstrap.sh"
-  config.vm.network "forwarded_port", guest: 5000, host: 5000 
-  config.vm.provider "virtualbox" do |v|
-    v.memory = 1024
+
+  config.vm.define "fedora", primary: true do |fedora|
+   fedora.vm.box = "fedora/30-cloud-base"
+   fedora.vm.provision "shell", path:"scripts/bootstrap.sh"
+   fedora.vm.network "forwarded_port", guest: 5000, host: 5000
+   fedora.vm.provider "virtualbox" do |v|
+     v.memory = 1024
   end
 end
+
+  config.vm.define "ubuntu" do |ubuntu|
+   ubuntu.vm.box = "ubuntu/bionic64"
+   ubuntu.vm.provision "shell", path:"scripts/bootstrap.sh"
+   ubuntu.vm.network "forwarded_port", guest: 5000, host: 5000
+   ubuntu.vm.provider "virtualbox" do |v|
+     v.memory = 1024
+  end
+end
+
+  config.vm.define "stage" do |stage|
+    stage.vagrant.plugins = "libxml2-dev"
+    stage.vagrant.plugins = "vagrant-aws"
+    stage.vm.box = "dummy"
+    stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
+    stage.vm.provider :aws do |aws, override|
+    stage.vm.synced_folder ".", "/vagrant", 
+      disabled: false
```

Comment:
Is it required? Isn't it enabled by default?

Reply:
For running the bootstrap.sh I moved this line to the start of the file and now is working, when I deleted the line it didn't work